_layouts/default.html

```diff
@@ -4,7 +4,7 @@
   <body class="grid-block vertical body page-landing">
     {% include icons.html %}
     {% include header.html %}
-    <main class="grid-block grid-container-lg padding-sm">
+    <main class="grid-block grid-container-lg">
```

Comment:
Having both padding and `.grid-container-lg` on this element is still making the content narrower than it is on master, like this: https://zippy.gfycat.com/LeanSadBighorn.webm

I think you can solve this by putting `.grid-container-lg` on the `.main-card` element instead. You'll have to adjust your padding styles too.